addons/medical_feedback/functions/fnc_effectBloodVolumeIcon.sqf

```diff
@@ -0,0 +1,46 @@
+#include "script_component.hpp"
+/*
+ * Author: 10Dozen
+ * Handles the blood volume icon
+ *
+ * Arguments:
+ * 0: Enable <BOOL>
+ * 1: Intensity 0...6 <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [true, 4] call ace_medical_feedback_fnc_effectBloodVolumeIcon
+ *
+ * Public: No
+ */
+params ["_enable", "_intensity"];
+
+disableSerialization;
```

Comment:
This command does nothing in unscheduled environment.

Reply:
Removed